tests/test_views.py

```diff
@@ -1264,25 +887,6 @@ def test_aliascontent_add_view_valid_data(self):
         alias_content = alias.contents.first()
         self.assertEqual(alias_content.name, 'test alias')
 
-    def test_category_change_view(self):
```

Comment:
I think that this test is still required because it is checking that the category title is shown correctly for different languages, this is built into django-parler and it's something that we should check still.